search/util.go

```diff
@@ -161,7 +192,14 @@ func (f FieldTermSynonymMap) MergeWith(fts FieldTermSynonymMap) {
 	}
 }
 
-const FieldTermSynonymMapKey = "_field_term_synonym_map_key"
-
-const SearcherStartCallbackKey = "_searcher_start_callback_key"
-const SearcherEndCallbackKey = "_searcher_end_callback_key"
+// BM25 specific multipliers which affect the scoring of a document.
+//
+// BM25_b - how much does a doc's field length affect the score
+// BM25_k1 - how much can the term frequency affect the score
+var BM25_k1 float64 = 1.2
+var BM25_b float64 = 0.75
```

Comment:
We can discuss this separately - how did you arrive at these defaults for b and k1? Could you pls add some commentary around that?

Reply:
these are defaults as per elastic search docs. i've added links to the blog and docs as a code comment.